impeller/display_list/display_list_dispatcher.cc

```diff
@@ -166,10 +219,27 @@ void DisplayListDispatcher::setColorFilter(
   // Needs https://github.com/flutter/flutter/issues/95434
   if (filter == nullptr) {
     // Reset everything
+    paint_.color_filter = std::nullopt;
     return;
   }
   switch (filter->type()) {
-    case flutter::DlColorFilterType::kBlend:
+    case flutter::DlColorFilterType::kBlend: {
+      auto dl_blend = filter->asBlend();
+
+      auto blend_mode = ToBlendMode(dl_blend->mode());
+      if (!blend_mode.has_value()) {
+        UNIMPLEMENTED;
```

Comment:
Does this need a TODO / issue? Or is it already covered by 95434?

Reply:
This would be https://github.com/flutter/flutter/issues/100086 -- added a TODO here.